main.py

```diff
@@ -1,17 +1,168 @@
-"""
-This is a starter file to get you going. You may also include other files if you feel it's necessary.
+import numpy as np
+import matplotlib.pyplot as plt
+import torch
+import torchvision
 
-Make sure to follow the code convention described here:
-https://github.com/UWARG/computer-vision-python/blob/main/README.md#naming-and-typing-conventions
+import torch.nn as nn
+import torch.optim as optim
+from torch import flatten
+from torch.nn import LogSoftmax
+import torch.nn.functional as F
+import torchvision.transforms as transforms
 
-Hints:
-* The internet is your friend! Don't be afraid to search for tutorials/intros/etc.
-* We suggest using a convolutional neural network.
-* TensorFlow Keras has the CIFAR-10 dataset as a module, so you don't need to manually download and unpack it.
-"""
+transform = transforms.Compose(
+    [transforms.ToTensor(), transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))
+     ])
 
-# Import whatever libraries/modules you need
+# this is our test set
+test = torchvision.datasets.CIFAR10(root='./data', train=False,
```

Comment:
Nit: Try using a better variable name. For example `test_dataset`. (You don't need to fix anything here, just smth to think about).